Exiled.Events/EventArgs/StartingSpeakerEventArgs.cs

```diff
@@ -41,10 +41,20 @@ public StartingSpeakerEventArgs(Player player, Room room, float apDrain, bool is
         /// </summary>
         public Room Room { get; }
 
+        /// <summary>
+        /// Gets or sets the amount of auxiliary power required to interact with a tesla gate.
```

Comment:
![](https://cdn.discordapp.com/emojis/585839489881079818.gif)

Reply:
Ah yeah, I copy pasted that property for all events and I've forgot to change the description.